src/Ampersand/PatchHelper/Helper/Magento2Instance.php

```diff
@@ -119,6 +102,107 @@ public function __construct(string $path)
             $libPath = sanitize_filepath($dirList->getRoot(), $libPath) . '/';
             $this->listOfPathsToLibrarys[$libPath] = $lib;
         }
+
+        $this->prepareThemes();
+    }
+
+    /**
+     * Prepare theme configuration and a list of paths to ignore in the scanning for hyva projects
+     *
+     * @link https://github.com/AmpersandHQ/ampersand-magento2-upgrade-patch-helper/issues/75
+     *
+     * As I understand it the reqs are
+     * - Only include frontend themes that have a Hyva/ root
+     * - We can just ignore file changes to theme files from non Hyva themes, don't need to check them
+     *
+     * @TODO hyva_theme_fallback/general/theme_full_path may require re-adding back in some excluded themes
```

Comment:
Any configured fallback theme should probably be treated as a regular Luma theme that is configured for a store view.

Reply:
The new approach we're not excluding any themes from the scan, so no need to be concerned with handling fallback specifically as it'll be a theme in and of itself.